packages/react-native/src/Authenticator/common/DefaultContent/styles.ts

```diff
@@ -12,7 +12,7 @@ export const getDefaultStyle = ({
       paddingHorizontal: space.small,
     },
     buttonPrimary: {
-      marginHorizontal: space.medium,
+      marginHorizontal: space.small,
       marginVertical: space.small,
```

Comment:
My suggestion might break snapshots:
```suggestion
      margin: space.small,
```